tutorials/quickstart/Solution/Messages/Messages.csproj

```diff
@@ -4,6 +4,6 @@
     <LangVersion>7.1</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.0.1" />
+    <PackageReference Include="NServiceBus" Version="7.*" />
```

Comment:
Do we really want this?

Reply:
Yeah, we always want the QuickStart using the latest and greatest NServiceBus.